tests/e2e/cases/E2eTest.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Inpsyde\CodingStandard\Tests\cases;
```

Comment:
Is this correct?

Reply:
In theory, yes according to this: https://github.com/inpsyde/php-coding-standards/blob/2385ad21a6cc92c084316557ee4f6b8f29228b25/composer.json#L51-L54

But at that point we should also change this: https://github.com/inpsyde/php-coding-standards/blob/2385ad21a6cc92c084316557ee4f6b8f29228b25/tests/unit/cases/FixturesTest.php#L29

But I thnk it would be better to setup the autoload as

```
    "Inpsyde\\CodingStandard\\Tests\\": [
        "tests/src/",
        "tests/unit/cases/",
        "tests/e2e/cases/"
    ]
```

And the use `namespace Inpsyde\CodingStandard\Tests` in all case classes.

Reply:
Thanks a lot! I slipped it out during refactoring, fixed it here - [`0aae3e5` (#86)](https://github.com/inpsyde/php-coding-standards/pull/86/commits/0aae3e5c922470f0aa7691514a5ca4e668a702ca).